doc/source/getting_started/docker.rst

```diff
@@ -124,3 +129,123 @@ with the `-np` switch.  For example:
 For additional command line arguments please see the ansys
 documentation at `ANSYS help <https://ansyshelp.ansys.com>`_.  Also,
 be sure to have the appropriate license for additional HPC features.
+
+Using ``--restart`` policy with MAPDL products
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, MAPDL creates a ``LOCK`` file at the working directory when starts
+and deletes it if it exits normally. This file is used to avoid overwriting the
+files such as database files (``db``) or result files (``rst``) when starting
+MAPDL after an abnormal termination.
```

Comment:
```suggestion
By default, MAPDL creates a ``LOCK`` file in the working directory when it starts
and deletes this file if it exits normally. The file is used to avoid overwriting files
such as database (DB) files or result (RST) files when starting MAPDL after an
abnormal termination.
```